packages/widget-chat/src/components/activity-list/index.js

```diff
@@ -0,0 +1,58 @@
+import React, {Component, PropTypes} from 'react';
+import ActivityItem from '../activity-item';
+
+class ActivityList extends Component {
+  shouldComponentUpdate() {
+    return false;
+  }
+
+  renderActivityItems(activities) {
+    return activities.map((activity) =>
+      <ActivityItem activity={activity} key={activity.id} />
+    );
+  }
+
+  render() {
+    const activities = this.renderActivityItems(this.props.activities);
+    return (
+      <div>
+        <ul>
+          {activities}
+        </ul>
+      </div>
+    );
+  }
+}
+
+ActivityList.propTypes = {
+  activities: PropTypes.array,
+  user: PropTypes.object
+};
+
+ActivityList.defaultProps = {
+  activities: [
```

Comment:
Thinking toward the future, it may make sense for the reducer to receive an `activity` and shape like a `message` from the developer api (with additional fields as needed to implement the behaviors the developer api doesn't support).